src/components/button/button.css

```diff
@@ -0,0 +1,64 @@
+@import '../../styles/colors.css';
+@import '../../styles/spacing.css';
+@import '../../styles/typography.css';
+
+.interactivity {
+  cursor: pointer;
+}
+
+.typography {
+  font-family: var(--mavenlink-type-font-family);
```

Comment:
Since we allow overrides via the CSS API, I think this is fine. 

We are also providing only 1 point of specificity so it can be easily cascaded away or `!important`.